dlp/api/Snippets/CalculateKAnonymityOnDataset.cs

```diff
@@ -0,0 +1,104 @@
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_k_anonymity_with_entity_id]
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Newtonsoft.Json;
+
+public class CalculateKAnonymityOnDataset
+{
+    public static DlpJob CalculateKAnonymitty(
+        string projectId,
+        string datasetId,
+        string sourceTableId,
+        string outputTableId)
+    {
+        // Construct the dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the k-anonymity config by setting the EntityId as user_id column
+        // and two quasi-identifiers columns.
+        var kAnonymity = new PrivacyMetric.Types.KAnonymityConfig
+        {
+            EntityId = new EntityId
+            {
+                Field = new FieldId { Name = "Name" }
+            },
+            QuasiIds =
+            {
+                new FieldId { Name = "Age" },
+                new FieldId { Name = "Mystery" }
+            }
+        };
+
+        // Construct risk analysis job config by providing the source table, privacy metric
+        // and action to save the findings to a BigQuery table.
+        var riskJob = new RiskAnalysisJobConfig
+        {
+            SourceTable = new BigQueryTable
+            {
+                ProjectId = projectId,
+                DatasetId = datasetId,
+                TableId = sourceTableId,
+            },
+            PrivacyMetric = new PrivacyMetric
+            {
+                KAnonymityConfig = kAnonymity,
+            },
+            Actions =
+            {
+                new Google.Cloud.Dlp.V2.Action
+                {
+                    SaveFindings = new Google.Cloud.Dlp.V2.Action.Types.SaveFindings
+                    {
+                        OutputConfig = new OutputStorageConfig
+                        {
+                            Table = new BigQueryTable
+                            {
+                                ProjectId = projectId,
+                                DatasetId = datasetId,
+                                TableId = outputTableId
+                            }
+                        }
+                    }
+                }
+            }
+        };
+
+        // Construct the request by providing RiskJob object created above.
+        var request = new CreateDlpJobRequest
+        {
+            ParentAsLocationName = new LocationName(projectId, "global"),
+            RiskJob = riskJob
+        };
+
+        // Send the job request.
+        DlpJob response = dlp.CreateDlpJob(request);
+
+        return response;
```

Comment:
The [java snippet](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/fafc6dca9dad437b9de9062d8f54168396fb0109/dlp/snippets/src/main/java/dlp/snippets/RiskAnalysisKAnonymityWithEntityId.java#L155) unpacks and prints the results. This can be helpful. 
Any constraints on including that?

Reply:
Initially we were waiting for the job to get completed but as per Amanda's suggestion we decided to remove the waiting time in the sample because job might take more amount of time to get completed which leads to timeout in test suite. That is why we have not waited for the job to get completed in other C# samples as well.